client/package.json

```diff
@@ -11,7 +11,8 @@
     "test:watch": "web-test-runner --watch",
     "build": "rimraf dist && rollup -c rollup.config.js",
     "start:build": "web-dev-server --root-dir dist --app-index index.html --open",
-    "start": "web-dev-server"
+    "start": "web-dev-server",
+    "prepare": "cd .. && husky install client/.husky"
```

Comment:
Wondering what husky is; I think I've seen it block some commits. If this command should be mentioned in the client dev docs, please add it in a separate PR